Assets/Scripts/Entity/Enemy/Enemy.cs

```
@@ -51,6 +51,8 @@ public override void ChangeMaxHealthBy(int amount)
      */
     public override void ChangeHealthBy(int amount)
     {
+        if (isDead) return;
```

Fail Fast....I like dis. Jazy approves